### DISCUSSION TITLE
Using -Fuzz to remove Transparency Halo

### DISCUSSION THREAD

**Discussion by starzar** [ImageMagick/ImageMagick#3923] [Help]
How to use the -fuzz command to remove Transparency Halo?
Results for the below command are not use-able.
 
command -
`magick test_opaque.png -transparent #000000 -fuzz 20% -fill white -opaque black test_alpha.png` 

**_Original Image_**
![test_opaque](https://user-images.githubusercontent.com/38550930/125260199-b7ac2100-e31d-11eb-94d3-9c16565f3d22.png)

**_Results-_** 
![fuzz_1](https://user-images.githubusercontent.com/38550930/125260270-cc88b480-e31d-11eb-8e17-ac26682f678c.jpg)

**Comment by snibgo** [comment]
What version of IM, on what platform?

Your command includes "-fill white -opaque black". This turns black pixels into white. When used with a fuzz, it turns pixels that are _close_ to black into white. Do you want to do that?

Is "Krita" an example of what you are trying to do? This doesn't seem to turn black pixels into white. Assuming you show it flattened on a #a22 background, it seems to use the lightness of the grayscale image as opacity, so a pixel that is 25% gray also becomes 25% opaque. Wndows CMD command:
```
magick test_opaque.png ( +clone ) -compose CopyOpacity -composite out.png
```
If the "Krita" result is what you want, please show the actual image, not flattened against a background.
